### DISCUSSION TITLE
小白如何选择笔记本电脑 - HAUE CS WIKI

### DISCUSSION THREAD

**Discussion by giscus** [Doge2077/haue-cs-wiki#7] [General]
# 小白如何选择笔记本电脑 - HAUE CS WIKI

http://127.0.0.1:8000/%E6%96%B0%E6%89%8B%E5%85%A5%E9%97%A8/%E5%B0%8F%E7%99%BD%E5%A6%82%E4%BD%95%E9%80%89%E6%8B%A9%E7%AC%94%E8%AE%B0%E6%9C%AC%E7%94%B5%E8%84%91/

**Comment by Doge2077** [comment]
老司机带带我，我要学编程🚕
